doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,90 @@
+=========
+Denoising
+=========
+
+This tutorial walks through the steps for denoising of diffusion-weighted MR images.
+There can be multiple denoising techniques.
+
+First you need to download the desired data.
+
+------------------------
+Downloading the Data Set
+------------------------
+
+You can view all the datasets using the command::
+
+    dipy_fetch list
+
+To fetch some specific dataset, you can use the following command::
+
+    dipy_fetch specific_dataset
+
+For example, to download sherbrooke_3shell dataset, you can use the following
+command::
+
+    dipy_fetch sherbrooke_3shell
+
+-------------------------
+Denoising using Local PCA
+-------------------------
+
+PCA-based denoising algorithms are effective denoising methods because they explore 
+the redundancy of the multi-dimensional information of diffusion-weighted datasets
+
+We are using isbi2013_2shell dataset. You can also use your own data.
+
+Following workflow requires input_files Path to the input volumes, bvalues_files Path
+to the bvalues files and bvectors_files Path to the bvectors files.
+
+Run the following workflow::
+
+    dipy_denoise_lpca /.dipy/isbi2013/phantom64.nii.gz /.dipy/isbi2013/phantom64.bval /.dipy/isbi2013/phantom64.bvec
+
+This workflow will generate a denoised volume (default: dwi_lpca.nii.gz) in the
+input directory by default.
+
+-----------------------------------
+Denoising using Marcenko-Pastur PCA
+-----------------------------------
+
+We are using cfin_multib dataset. You can also use your own data.
+
+Create an out_dir folder (eg: denoise_mppca_output)::
+
+    mkdir denoise_mppca_output
+
+Run the following workflow::
+
+    dipy_denoise_mppca /.dipy/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii --patch_radius 10 --out_dir "denoise_mppca_output"
+
+This workflow will generate a denoised volume and a sigma volume 
+in the speified output directory.
+
+-----------------------
+Denoising using NLMEANS
+-----------------------
+
+We are using sherbrooke_3shell dataset. You can also use your own data.
+
+Following workflow requires input_files Path to the input volumes.
+You can use optional arguments as well.
```

Comment:
Please, re-word the same way as I suggested above. `input_files` is a positional argument, so you usually do not refer it by name. You can say:

In order to run the NLMEANS denoising method, we need to specify the location of the diffusion data file, followed by the optional arguments. In this case, we will be specifying the sigma and patch radius values. So we will run the command as::

Same comment as above applies concerning the data path.

Reply:
Reworded in d49dc04